app/v2/models/request/createUkPropertyPeriodSummary/ukNonFhlProperty/UkNonFhlPropertyExpensesRentARoom.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.request.createUkPropertyPeriodSummary.ukNonFhlProperty
+
+import play.api.libs.json.{Json, OFormat}
+
+case class UkNonFhlPropertyExpensesRentARoom(amountClaimed: Option[BigDecimal])
```

Comment:
Can FHL and Non-FHL share an ExpensesRentARoom object rather than having the same object twice with diffrent names